### ISSUE TITLE
When user navigates to the Informational button icon present inside the 'Source code' popup, Narrator fails to announce the appeared tooltip information

### ISSUE THREAD

**Issue by ayushjai19** [microsoft/WinUI-Gallery#2088]
### Which version of the app?

WinUI 3 Gallery

### Description

Environment Details:
OS Version: 25H2
OS Build: 26200.6725
App: WinUI 3 Gallery
App Version: 2.7.0.0 (SDK 1.8)
AT: Screen reader

Pre-Requisite:
Turn on Narrator with shortcut key "win+ctrl+Enter" key.

Repro Steps:
Launch the WinUI 3 Gallery app.
Navigate to search box using tab key.
Type 'Resources' in search box and hit enter.
Navigate to 'Resources' card and activate it.
Navigate to 'Source' button and expand it.
Now observe the informational icon present in source code popup 
Now listen to the screen reader announcement
Actual Result:
When user navigates to the Informational button icon present inside the 'Source code' popup, Narrator fails to announce the appeared tooltip information. It simply announcing as "Info, button".

Note: Same issue is observed in Entire Application.

Expected Result:
When user navigates to the Informational button icon present inside the 'Source code' popup, Narrator should announce the displayed tooltip information along with the <label name>, <control type>.

User Impact:
Users with visual impairments who rely on screen reader will face difficulties, if screen reader fails to announce the displayed tooltip information for informational button.

MAS Reference Link: [MAS 4.2.1 – Object Information](https://aka.ms/MAS4.2.1)

Trap ID: [1.4 - Uncomprehended Element](https://aka.ms/UncomprehendedElement)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

### Windows version

Windows 11 24H2 (26100)
